<launch>

  <node pkg="y1_controller" type="y1_controller.py" name="y1_arm" output="screen"> 
    <!-- CAN ID -->
    <param name="arm_can_id" value="can0"/>
    <!-- arm status and joint state feedback frequency (HZ) -->
    <param name="arm_feedback_rate" value="200"/>
    <!-- arm control type: leader_arm(remote master arm), follower_arm(remote slave arm),
                           normal_arm(normal control arm) -->
    <param name="arm_control_type" value="normal_arm"/>
    <!-- 0: nothing, 1: gripper, 2: teaching pendant -->
    <param name="arm_end_type" value="3"/>
    <!-- whether to enable robotic arm -->
    <param name="auto_enable" value="true"/>

    <!-- end pose control arm -->
    <param name="arm_end_pose_control_topic" value="/y1/arm_end_pose_control"/>
    <!-- joint position control arm -->
    <param name="arm_joint_position_control_topic" value="/y1/arm_joint_position_control"/>
    <!-- end pose and joint position feedback -->
    <param name="arm_joint_state_topic" value="/y1/arm_joint_state"/>
    <!-- joint motor status feedback -->
    <param name="arm_status_topic" value="/y1/arm_status"/>
  </node>

</launch>